### ISSUE TITLE
Error after connecting BitBucket $pushAll is depreciated

### ISSUE THREAD

**Issue by hongkongkiwi** [Strider-CD/strider#1059]
Hi,

When using Strider from a docker and trying to connect BitBucket (after pushing allow) I receive this error:

Looks like strider is using `$pushAll` which is now depreciated and removed from the latest mongodb. Any workaround or fixes?

```
Connect
500 MongoError: Unknown modifier: $pushAll
   at Function.MongoError.create (/data/node_modules/mongoose/node_modules/mongodb-core/lib/error.js:31:11)
   at toError (/data/node_modules/mongoose/node_modules/mongodb/lib/utils.js:139:22)
   at /data/node_modules/mongoose/node_modules/mongodb/lib/collection.js:1059:67
   at /data/node_modules/mongoose/node_modules/mongodb-core/lib/connection/pool.js:469:18
   at process._tickCallback (internal/process/next_tick.js:176:11)
```

After some research I find that this issue is caused by Mongoose and fixed by upgrading mongoose to version 5 or later. Can we get the package.json updated?

**Comment by xgalen** [comment]
As described [here](https://github.com/Automattic/mongoose/blob/b1ffbc1ff22000ecd8f39ffc04c7075337e2a1cb/migrating_to_5.md), not only changing the version is enough, the code has to be migrated too. A pull-request is welcome :)

**Comment by knownasilya** [comment]
So they changed v4 in a breaking way? Or is v5 being installed when it shouldn't be?

**Comment by knownasilya** [comment]
@hongkongkiwi what version is in `/data/node_modules/mongoose/node_modules/mongodb/package.json`?

**Comment by guruparan** [comment]
I had the same issue and i tried with latest version 5.2.6 and it worked fine

**Comment by knownasilya** [comment]
Latest of mongoose?

**Comment by guruparan** [comment]
Yes mongoose 5.2.6

**Comment by knownasilya** [comment]
I've released v1.10.1 to use mongoose v5. This uses the new db_uri format, i.e. you need to specify a port now.
